Tests/JWSRSATests.swift

```diff
@@ -149,7 +128,19 @@ class JWSRSATests: RSACryptoTestCase {
         let verifier = Verifier(verifyingAlgorithm: algorithm, publicKey: publicKeyAlice2048!)
 
         XCTAssertTrue(secondJWS.isValid(for: verifier!))
-        XCTAssertEqual(String(data: jws.header.data(), encoding: .utf8), "{\"alg\":\"\(algorithm.rawValue)\"}")
+        if withKid {
+            let algKidHeader = "{\"alg\":\"\(algorithm.rawValue)\",\"kid\":\"kid\"}"
+            let kidAlgHeader = "{\"kid\":\"kid\",\"alg\":\"\(algorithm.rawValue)\"}"
```

Comment:
I think you could make these string literals a little cleaner to read by using [extended string delimiters](https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html#ID606). Your call whether it's worth the trade off for noisier string interpolation.

Something like 
```swift
let algKidHeader = #"{"alg":"\#(algorithm.rawValue)","kid":"kid"}"#
```

Reply:
Agreed, but since we have lots and lots of such strings throughout the project, I tracked https://github.com/airsidemobile/JOSESwift/issues/202.